src/modules/connections/__tests__/ConnectionService.test.ts

```diff
@@ -130,6 +147,26 @@ describe('ConnectionService', () => {
       expect(aliasDefined.alias).toBe('test-alias')
       expect(aliasUndefined.alias).toBeUndefined()
     })
+
+    it('returns a connection record with mediator information', async () => {
+      expect.assertions(1)
+
+      const mockMediatorFind = mediationRepository.findById as jest.Mock<Promise<MediationRecord>, [string]>
```

Comment:
We have a new util method in `src/__tests__/helpers.ts`

```suggestion
      const mockMediatorFind = mockFunction(mediationRepository.findById)
```